fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py

```diff
@@ -1046,6 +1075,13 @@ def encode_list(
                     f"both padding token and padding id are None, but padding length is {max_len}. It's possible that it was set for truncation alone."
                 )
 
+        # Check if we have any unknown tokens in our input
+        if self.count_unknowns(merged_encoding) > 0:
```

Comment:
Do you think it'll be time consuming to count in each encoding?
If so maybe we should consider an option `on_unkown="ignore" / "skip" / "pass" , etc`, and make skipping the counting optional

Just a thought

Reply:
It's just a single pass over the ID list - should be negligible to the stuff going on during encode call.